src/python/seccomp.pyx

```diff
@@ -988,18 +1125,20 @@ cdef class SyscallFilter:
         free(req)
         return notify
 
-    def respond_notify(self, response):
+    def respond_notify(self, response, fd = None):
```

Comment:
Why do we need to support passing a file descriptor here?  I'm having difficulty understanding what value it belongs to this method.

Reply:
It enables the use case where the caller forks a process and installs a seccomp filter in its child process (and immediately gets an fd assigned). This child, now with the filter, sends this fd back to the caller (by any method), and the caller needs a way to respond to this particular fd. In the case of the `C` API, it's already like that. But for Python, it was causing an issue when the caller needed to respond and install the new fd (via addfd) and notify via this notify fd. The `fd = None` case is kept for backward compatibility.